docs/BrandColors.stories.tsx

```diff
@@ -51,7 +51,7 @@ export const CSS: Story = {
   },
 };
 
-export const JS: Story = {
+export const JS = {
```

Comment:
Is this change necessary? The other Stories are typed can we either remove it from all stories or add it here?
```suggestion
export const JS: Story = {
```